### DISCUSSION TITLE
LUTs

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#753] [General]
Hello team,

It has been with this Labs firmware. I wonder if you can have an in-house LUT for the log base 400. It would be great if it is coming in-house since you know the secret "sauce".

**Comment by dnewman-gpsw** [comment]
LOG base 400 is the same as GP-Log so you can use those LUTs.  Linked at the bottom of this page: https://community.gopro.com/s/article/10-Bit-Log-Encoding?language=en_US

version 2.00 LUTs are for oMLOGB=400oMWIDE=1
version 1.20 LUTs are for oMLOGB=400 without wide gamut enabled.

**Reply by dnewman-gpsw** [reply]
It produces a Rec709 image will a GoPro-like look. However, LUTs can't do tone-mapping, and Natural/GoPro Color are tone-mapped.  So a LUT can never much the in-camera look exactly, but it close to the base look.

**Reply by dnewman-gpsw** [reply]
Flat + WIDE=1 using the GP-Log LUT would have too much contrast, but you convert Log 113 (Flat) to Log 400 (GP-Log) before the LUT.  Math is here https://www.desmos.com/calculator/srvh1rkeur

**Comment by tetrahydra** [comment]
Thank you David. I thought these only worked for GP12. I am staying on with GP11 until GoPro has a bigger sensor.

**Reply by dnewman-gpsw** [reply]
GPLog was developed using Labs on HERO11, so they work on both.
